mindmeld/app.py

```diff
@@ -171,6 +172,20 @@ def _decorator(func):
 
         return _decorator
 
+    def auto_fill(self, **kwargs):
+        """Creates a flow to fill missing entities"""
+
+        def _decorator(func):
+            name = kwargs.pop("name", func.__name__)
+            entity_form = kwargs.pop("entity_form", None)
+            assert entity_form, "Entity Form cannot be empty."
```

Comment:
If you need to verify the type of some variable, it's better to raise a `TypeError` rather than an assertion. You may also want to use `isinstance` to see what the parameter actually is, though I don't know exactly what you expect here.
```python
if not isinstance(entity_form, (tuple, list, MyType)):
    raise TypeError(...)
```
The reason is twofold. One is that a TypeError simply conveys more explicit information ("hey dude, the type is wrong!") and, though this doesn't happen often, if Python is invoked with the `-O` option to optimize bytecode, the interpreter strips assertions.

Reply:
The idea here was to ensure that the form isn't empty. Do you think TypeError goes for that as well?